Dockerfile.compute-node

```diff
@@ -784,7 +784,7 @@ FROM build-deps AS pg-partman-build
 COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
 
 ENV PATH="/usr/local/pgsql/bin/:$PATH"
-RUN wget https://github.com/pgpartman/pg_partman/archive/refs/tags/v5.0.1.tar.gz -O pg_partman.tar.gz && \
+RUN wget https://github.com/pgpartman/pg_partman/archive/refs/tags/v4.7.3.tar.gz -O pg_partman.tar.gz && \
```

Comment:
checksum will be different for this archive

Reply:
Good catch, thanks